sdk/playwrighttesting/arm-playwrighttesting/review/arm-playwrighttesting.api.md

```diff
@@ -12,11 +12,7 @@ import { SimplePollerLike } from '@azure/core-lro';
 
 // @public
 export interface Account extends TrackedResource {
-    readonly dashboardUri?: string;
-    readonly provisioningState?: ProvisioningState;
-    regionalAffinity?: EnablementStatus;
-    reporting?: EnablementStatus;
-    scalableExecution?: EnablementStatus;
+    properties?: AccountProperties;
```

Comment:
Is this caused by turning of the flatten ?

Reply:
yes